seqr/views/apis/data_manager_api_tests.py

```diff
@@ -1309,27 +1351,43 @@ def test_validate_callset(self, mock_subprocess):
         response = self.client.post(url, content_type='application/json', data=json.dumps(body))
         self.assertEqual(response.status_code, 200)
 
+    @mock.patch('seqr.views.utils.airtable_utils.is_google_authenticated', lambda x: True)
+    @responses.activate
     def test_get_loaded_projects(self):
         url = reverse(get_loaded_projects, args=['WGS', 'SV'])
         self.check_pm_login(url)
 
         response = self.client.get(url)
         self.assertEqual(response.status_code, 200)
-        self.assertDictEqual(response.json(), {'projects': [
-            {'dataTypeLastLoaded': '2018-02-05T06:31:55.397Z', 'name': 'Non-Analyst Project', 'projectGuid': 'R0004_non_analyst_project'},
-        ]})
+        self.assertDictEqual(response.json(), {'projects': [{**PROJECT_OPTION, 'dataTypeLastLoaded': '2018-02-05T06:31:55.397Z'}]})
 
         response = self.client.get(url.replace('SV', 'MITO'))
         self.assertEqual(response.status_code, 200)
-        self.assertDictEqual(response.json(), {'projects': [
-            {'dataTypeLastLoaded': None, 'name': 'Non-Analyst Project', 'projectGuid': 'R0004_non_analyst_project'},
-        ]})
+        self.assertDictEqual(response.json(), {'projects': [PROJECT_OPTION]})
 
         # test data manager access
         self.login_data_manager_user()
         response = self.client.get(url)
         self.assertEqual(response.status_code, 200)
 
+        # test with airtable filter
+        responses.add(
+            responses.GET, 'https://api.airtable.com/v0/app3Y97xtbbaOopVR/PDO', json=AIRTABLE_PDO_RECORDS, status=200,
+        )
+        snv_indel_url = url.replace('SV', 'SNV_INDEL')
+        response = self.client.get(snv_indel_url)
+        self.assertEqual(response.status_code, 200)
+        self.assertDictEqual(response.json(), {'projects': [EMPTY_PROJECT_OPTION, PROJECT_SAMPLES_OPTION]})
+        self.assert_expected_airtable_call(
+            call_index=0, filter_formula="OR(PDOStatus='Methods (Loading)',PDOStatus='On hold for phenotips, but ready to load')",
+            fields=['PassingCollaboratorSampleIDs', 'SeqrIDs', 'SeqrProjectURL'],
+        )
+
+        # test projects with no data loaded are returned for any sample type
```

Comment:
@jklugherz this tests the `exclude` behavior you were asking about - changing the logic the way you suggested would cause the empty project to not be returned here